src/test_workflow/test_args.py

```diff
@@ -24,7 +24,7 @@ class TestArgs:
     def __init__(self):
         parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
         parser.add_argument("path", type=str, help="Location of build and bundle manifests.", default=".")
-        parser.add_argument("--test-manifest-path", type=str, help="Specify a test manifest path.")
+        parser.add_argument("test-manifest-path", type=str, help="Specify a test manifest path.")
```

Comment:
Should this be the first argument, then the location of build and bundle manifest path already has a default, so you won't have to do `./test.sh . test_manifest`, can just do `./test.sh test_manifest`.

Reply:
moved to first.